### PR TITLE
Fixed missing link for facebook conversion tracking

### PR DESCRIPTION
**Pull request by philipecosia**

### CODE REVIEW

**Review by axlwaii — APPROVED** [review]
:lock:

File: en.md [hunk 1]
```diff
@@ -89,11 +89,11 @@ We use several offerings of third-party providers to advertise our app on extern
 
 We use the following conversion tracking services in the app:
 
-- Facebook: [https://www.facebook.com/about/privacy](https://www.facebook.com/about/privacy). Facebook has undertaken to uphold the EU-US Privacy Shield.
+- Facebook Inc., 1601 S California Ave, Palo Alto, California 94304, USA; [http://www.facebook.com/policy.php](http://www.facebook.com/policy.php)
```
Comment (LostCrew) [inline en.md:92]: @philipecosia Any reason why you switched from HTTPS for HTTP?

Comment (lopis) [reply en.md:92]: We should definitely change it, but actually if you check the link, it automatically redirects to https.

Comment (LostCrew) [reply en.md:92]: Still, wild non-HTTPs links in prod allow for MITM attacks. We shouldn't be those people that facilitate them

Comment (Zissi) [reply en.md:92]: I didn't see that I'll fix it. Thanks for spotting

**Pull request merged**